src/ignoreme.rs

```diff
@@ -36,11 +36,7 @@ fn get_ignored(location: &PathBuf) -> Option<Vec<PathBuf>> {
     for x in all_set.difference(&ign_set) {
         output.push(x.to_owned());
     }
-    if output.is_empty() {
-        None
-    } else {
-        Some(output)
-    }
+    Some(output)
```

Comment:
I checked the file and didn't found any clue about an Option type other that `Some(output)`.
If I did not made any mistake in reading the code, why do we return an `Option` type if it's always `Some(output)`... ?

Reply:
yeah i think that's a good call, previously it returned an option because there may be nothing in Output, i'll change it